helpers/apolloClient.ts

```diff
@@ -33,24 +30,18 @@ const whiteList = ['/curriculum']
   }
 })()
 
-function createIsomorphLink() {
-  if (typeof window === 'undefined') {
-    const { SchemaLink } = require('@apollo/client/link/schema')
-    const { schema } = require('../graphql/schema')
-    return new SchemaLink({ schema })
-  } else {
-    const { HttpLink } = require('@apollo/client/link/http')
-    return new HttpLink({
-      uri: '/api/graphql',
-      credentials: 'same-origin'
-    })
-  }
+function createHttpLink() {
+  const { HttpLink } = require('@apollo/client/link/http')
+  return new HttpLink({
```

Comment:
if the import above works then we don't need this `creatHttpLink` helper function anymore